src/libertem/io/dataset/frms6.py

```diff
@@ -452,7 +452,7 @@ def get_diagnostics(self):
         global_header = self._get_hdr_info()
         return [
             {"name": "Offset correction available and enabled",
-             "value": str(self._get_dark_frame() is not None)},
```

Comment:
@sk1p This was probably a merge artifact? What is correct here?

Reply:
No, change looks good - this was a fix for a bug that was caught as part of changing the `dist` tests to use the new cli

Reply:
Thx! :-)